manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: elasticsearch-operator.v4.1.1
+  name: elasticsearch-operator.v4.1.2
```

Comment:
@jwforres are we supposed to have the version in the name here?  seems odd to me.